themes/default/layouts/essay/single.html

```diff
@@ -16,6 +16,11 @@
       <div class="content">
         {{ .Content }}
         {{ partial "page-bibliography.html" . }}
+        {{ if eq .Section "documentation" }}
```

Comment:
Whenever I add custom code to a Quire project, rather than doing it in the `themes/default/layouts/` directory, I like to make a copy of the file and add it to the `layouts` directory at the root of the project. This then acts as an override to to the version in `themes/default/`. Then I add a comment at the top of the file explaining what's unique about it, what the override is. this makes it easier to update the theme down the road without losing all the customizations. You can see an example of this in the https://github.com/thegetty/quire-docs/blob/main/layouts/_default/baseof.html file.

Could you do that here?

Reply:
Not sure if I integrated your suggestion in an intelligent way, perhaps I misunderstood your (clever) suggestion :thinking: